app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,34 @@
+class SessionsController < ApplicationController
+  skip_before_action :authorized, only: [:new, :create, :welcome]
+  
+  def new
+  end
```

Comment:
Use single line syntax for empty method
```suggestion
  def new; end
```